content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -25,201 +25,147 @@ colors). Each one of these products has its own inventory, sales, etc.
-  and add some lines that are specific to the dimension S, and
-  other lines specific to the color Red.
+- **Picture**: Every product variation can have its own primary picture.
 
-- **Pricing**: The default price of a product is computed using the
-  price of the product template and add the optional extra price on
-  each dimension of the variant. This way, variant prices are
-  easier to maintain since you don't have to set the price for
-  every variant. However, it's possible to create pricelist rules
-  to fix price per variants too.
+- **Other fields**: Many fields belong specifically to the product template. Updating these
+  fields automatically updates every variant as well. Examples include Income Account and Taxes.
 
-When should you avoid using variants?
--------------------------------------
+Configuring Variants
+====================
 
-Using variants may add a level of complexity on the way you use Odoo.
-You should consider using variants only if you need it to reduce the
-complexity of managing lots of products that are similars.
+Activating the variant feature
+------------------------------
 
-As an example, importing your initial product catalog is more complex if
-you use variants. You can't just import a list of products, you must
-import product templates and all their related variations.
+Before using product variants, the setting needs to be activated. To do so, you must go to the
+Sales app. Select :menuselection:`Configuration --> Settings`, locate the **Products Variants**
+line via scrolling or the search bar at the top of the page.
 
-In addition to that, you should also carefully select the dimensions
-that you manage as separate product templates and those as variants. As
-an example, a company having these products:
+.. image:: variants/activating-variants-setting.png
+   :align: center
+   :alt: Activating Product Variants settings
 
-- Quality: T-Shirts, Polos, Shirts
+Creating Attributes
+-------------------
 
-- Color: Red, Blue
+Attributes need to be created before product variants can be set up in your database. Attributes
+can be accessed via :menuselection:`Configuration --> Attributes`. To create a new Attribute,
+click **Create**. First, choose an attribute name, such as “color” or “size.”
 
-- Size: S, M, L, XL
+Then, select a display type from Radio, Select, and Color, which determines how your product will
+be shown on your eCommerce page, PoS dashboard or Product Configurator. Radio lists your options
+in a bullet style list; Select creates a dropdown menu; and Color creates little squares to
+choose the options from.
 
-In such a use case, you could create 1 template with three dimensions of
-variants (Layout, T-Shirts, Polos). But, it's recommended to create two
-different product templates as T-shirts may highly differ from polos or
-shirts and customer expect to see these as two different products in the
-e-Commerce:
+.. image:: variants/display-types-configurator.png
```

Comment:
I actually had this screenshot here because I wanted to show how they actually appear when you activate the different display types since this is definitely something I really wish I had a visual of when I was starting out with the Sales app. I'd rather put a separate screenshot of the attribute window at the beginning.

Reply:
Both screenshots are useful I believe!